.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
+  "useTabs": false,
+  "printWidth": 80,
+  "tabWidth": 2,
```

Comment:
Would it be okay to keep it to 4 ? :) 
I don't mind tabs but a width of 2 is very hard on my eyes ^^